modules/user_accounts/test/user_accountsTest.php

```diff
@@ -332,7 +332,7 @@ function testAddNewUser()
 
     /**
      * Modifies a field on either the user account or my preferences page
-     * and checks that the modification was recorded in the database.
+     * and checks that the modification was updated on the front-end.
```

Comment:
github is highlighting the below `'user_accounts/my_preferences'`, line as a whitespace error.